lib/widgets/pages/login/login_email_page.dart

```diff
@@ -1,27 +1,74 @@
 import 'package:coffeecard/base/strings.dart';
 import 'package:coffeecard/cubits/login/login_cubit.dart';
+import 'package:coffeecard/data/repositories/shared/account_repository.dart';
+import 'package:coffeecard/service_locator.dart';
+import 'package:coffeecard/widgets/components/dialog.dart';
 import 'package:coffeecard/widgets/components/entry/login/login_cta.dart';
 import 'package:coffeecard/widgets/components/entry/login/login_email_text_field.dart';
+import 'package:coffeecard/widgets/components/entry/register/email_body.dart';
+import 'package:coffeecard/widgets/components/scaffold.dart';
 import 'package:coffeecard/widgets/pages/login/login_page.dart';
 import 'package:coffeecard/widgets/routers/register_flow.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
 
-void _changeRoute(BuildContext context) {
-  BlocProvider.of<LoginCubit>(context).clearError();
-  Navigator.of(context).push(RegisterFlow.route);
-}
-
 class LoginEmailPage extends LoginPage {
   LoginEmailPage()
       : super(
           inputWidget: const LoginEmailTextField(),
           resizeOnKeyboard: true,
           ctaChildren: [
+            const LoginCTA(
```

Comment:
placed the 'forgot passcode' button on the email login screen. Not sure if it is correctly placed?

Reply:
- The spacing between CTAs should be 8.
- Preferably, the forgot button is in the passcode page  as either a numpad action or a CTA, see picture

![image](https://user-images.githubusercontent.com/21163286/158859108-448fb144-9587-4768-9fb5-54febbf3f158.png)